recipe/meta.yaml

```diff
@@ -17,13 +17,19 @@ requirements:
     - python
     - pip
     - notebook >=4.4.1
```

Comment:
There is in the host should be `jupyter_packaging >=0.10,<2`, see https://github.com/jupyter-widgets/ipywidgets/blob/master/python/widgetsnbextension/pyproject.toml
But I can't find the tag 3.5.2 in the upstream source so it's from master branch